marklogic/README.md

```diff
@@ -0,0 +1,98 @@
+# Agent Check: MarkLogic
+
+## Overview
+
+This check monitors [MarkLogic][1] through the Datadog Agent.
```

Comment:
```suggestion
This check monitors [MarkLogic][1] through the Datadog Agent. MarkLogic Server is a multi-model database designed to be a data hub for operational and analytical data.
```

Include a line about MarkLogic, maybe even overview of the types of metrics collected?